test/services/billing-accounts/change-address.service.test.js

```diff
@@ -0,0 +1,424 @@
+'use strict'
```

Comment:
Just as a general point, all of our tests call `ChangeAddressService.go()` with `agentCompany` and `contact`. Presumably some of these tests are passing in empty values but is it worth having at least one test that doesn't pass these in, just to confirm/document the defaulting of these to empty objects?

Reply:
Added a test and it made me think about what the service returns. So, also helped me clean things up a bit. Thank you!